code/modules/power/port_gen.dm

```diff
@@ -254,6 +254,13 @@
 	var/datum/gas_mixture/environment = loc.return_air()
 	if (environment)
 		environment.adjust_gas_temp(GAS_PHORON, phoron/10, temperature + T0C)
+	var/rads = 90 + (sheets + sheet_left)*1.5
+	for (var/mob/living/L in range(src, 10))
+		//should really fall with the square of the distance, but that makes the rads value drop too fast
+		//I dunno, maybe physics works different when you live in 2D -- SM radiation also works like this, apparently
+		L.apply_effect(max(20, round(rads/get_dist(L,src))), IRRADIATE, blocked = L.getarmor(null, "rad"))
+	explosion(loc, 3, 6, 12, 16, 1) //explosion about as big as a Supermatter shard, also produces radiation. Mars told us phoron is dangerous.
```

Comment:
Does this give an appropriate warning before violently Marsing the atmosphere? I think it should. Sirens and all.

Reply:
I will do this once I work out how to.